packages/contracts/scripts/write-version.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version } = require('../package.json')
+
+/**
+ * Generates a Typescript constant for the contracts package version which is used by the plugins package to determine what
+ * the correct version of the Sphinx library contracts is.
+ */
+const writeVersion = async () => {
+  const file = `export const CONTRACTS_LIRARY_VERSION = 'v${version}'\n`
+  process.stdout.write(file)
+}
+
+writeVersion()
```

Comment:
This just generates a TS constant for the contracts package version, so we can easily fetch it in the `plugins` package.